docs/fsharp/language-reference/basic-types.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Basic Types (F#)
+description: Discover the fundamental basic types that are used in the F# language.
+ms.date: 07/09/2018
+---
+# Basic types
+
+This topic lists the basic types that are defined in the F# language. These types are the most fundamental in F#, forming the basis of nearly every F# program. They are a superset of .NET primitive types.
+
+|Type|.NET type|Description|
+|----|---------|-----------|
+|`bool`|<xref:System.Boolean>|Possible values are `true` and `false`.|
```

Comment:
nit: if you want the link to display the namespace as before, you can add the following query parameter to the xref links:
?displayProperty=nameWithType

Reply:
I think I'll leave them without the `System.` prefix. I prefer it without them, actually.